src/js/components/clientMirror/ActionBar.js

```diff
@@ -1,5 +1,6 @@
 import React from 'react';
 import PropTypes from 'prop-types';
+import FaFloppyO from 'react-icons/lib/fa/floppy-o';
```

Comment:
Yes, I think we should Bootstrap icons as we are already using them: https://www.w3schools.com/icons/bootstrap_icons_glyphicons.asp

I can change that.